__tests__/app.test.js

```diff
@@ -5,6 +5,7 @@ const request = require("supertest")
 const testData = require("../db/data/test-data")
 const seed = require("../db/seeds/seed")
 const db = require("../db/connection")
+const sorted = require("jest-sorted")
```

Comment:
You shouldn't need to require in jest sorted - we actually need to add the set up code to our package.json - you can find this in the jest sorted docs!